.github/workflows/ci-build.yml

```diff
@@ -1,24 +1,25 @@
 name: ci-build
 
 on:
-  push:
-    paths:
-      - 'clients/**'
-      - 'common/**'
-      - 'explorer-api/**'
-      - 'gateway/**'
-      - 'integrations/**'
-      - 'mixnode/**'
-      - 'sdk/lib/socks5-listener/**'
-      - 'sdk/rust/nym-sdk/**'
-      - 'service-providers/**'
-      - 'nym-api/**'
-      - 'nym-outfox/**'
-      - 'tools/nym-cli/**'
-      - 'tools/nym-nr-query/**'
-      - 'tools/ts-rs-cli/**'
-      - 'Cargo.toml'
-      - '.github/workflows/ci-build.yml'
+  #push:
+  #  paths:
+  #    - 'clients/**'
+  #    - 'common/**'
+  #    - 'explorer-api/**'
+  #    - 'gateway/**'
+  #    - 'integrations/**'
+  #    - 'mixnode/**'
+  #    - 'sdk/lib/socks5-listener/**'
+  #    - 'sdk/rust/nym-sdk/**'
+  #    - 'service-providers/**'
+  #    - 'nym-api/**'
+  #    - 'nym-node/**'
+  #    - 'nym-outfox/**'
+  #    - 'nym-validator-rewarder/**'
+  #    - 'tools/**'
+  #    - 'Cargo.toml'
+  #    - 'Cargo.lock'
+  #    - '.github/workflows/ci-build.yml'
```

Comment:
why comment out and not delete?

Reply:
There is no good answer to that, let me delete that!